device/sycl/src/clusterization/component_connection.hpp

```diff
@@ -0,0 +1,32 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// SYCL library include(s).
+#include "traccc/sycl/utils/queue_wrapper.hpp"
+
+// Project include(s).
+#include "traccc/clusterization/detail/sparse_ccl.hpp"
+#include "traccc/edm/cell.hpp"
+#include "traccc/edm/cluster.hpp"
+
+// Vecmem include(s).
+#include <vecmem/memory/memory_resource.hpp>
+
+namespace traccc::sycl {
+
+/// Forward declaration of component connection function
+///
+void component_connection(
+    cluster_container_view clusters_view, cell_container_const_view cells_view,
+    const host_cell_container& cells_per_event,
```

Comment:
The function should either receive a `cell_container_const_view` or a `host_cell_container` parameter. The two describe the same container, so only one is necessary. (You can create a `cell_container_const_view` inside of the function from `host_cell_container` if you choose to pass the latter to the function.)

Reply:
Oh wow, I don't know how did that happen, of course, those are the same.